src/ui/summary-ui-handler.ts

```diff
@@ -345,6 +373,15 @@ export default class SummaryUiHandler extends UiHandler {
 
     this.candyShadow.setCrop(0, 0, 16, candyCropY);
 
+    if (this.friendshipShadow.visible) {
+      this.friendshipShadow.on("pointerover", () => (this.scene as BattleScene).ui.showTooltip("", `${i18next.t("pokemonSummary:friendship")}`, true));
+      this.friendshipShadow.on("pointerout", () => (this.scene as BattleScene).ui.hideTooltip());
```

Comment:
```suggestion
      this.friendshipShadow.on("pointerover", () => this.scene.ui.showTooltip("", `${i18next.t("pokemonSummary:friendship")}`, true));
      this.friendshipShadow.on("pointerout", () => this.scene.ui.hideTooltip());
```

Pretty sure this.scene is a BattleScene in UiHandlers. Just double check if possible, and might as well update it for the candy tooltip as well if it's the case